compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/symbols/BallerinaTypeSymbol.java

```diff
@@ -17,29 +17,23 @@
 package io.ballerina.compiler.api.impl.symbols;
 
 import io.ballerina.compiler.api.ModuleID;
-import io.ballerina.compiler.api.symbols.SimpleTypeSymbol;
 import org.wso2.ballerinalang.compiler.semantics.model.types.BType;
 import org.wso2.ballerinalang.compiler.util.CompilerContext;
 
 /**
- * Represents the built-in simple type descriptor.
+ * Represents type descriptors which do not fall in to any other type descriptor.
```

Comment:
What are the possible types that could fall on to this group?

Reply:
With the changes introduced in this PR, this becomes a sort of a fall back type symbol for any typedescs we may have missed. Currently the only one I can think of is the "dependent type". Plan is to get rid of this altogether for alpha.